all-unix-style.yml

```diff
@@ -135,10 +135,10 @@ properties:
         type: boolean
         title: Enable generation of a openpgp signed Chain of Trust artifact
```

Comment:
This line still mentions just openpg. I suspect just `s/openpgp//` is the appropriate fix.

Reply:
Done.